catalyst-gateway/bin/src/db/index/tests/mod.rs

```diff
@@ -0,0 +1,4 @@
+//! Integration tests of the `IndexDB` queries
+
+#[cfg(test)]
```

Comment:
Nitpick: I believe the attribute isn't needed here because the `tests` module already has it.

Reply:
Thank you for spotting on this. I removed this part.